src/tags_int.cpp

```diff
@@ -2164,6 +2164,32 @@ namespace Exiv2 {
         return os << value;
     }
 
+    std::ostream& printBitmask(std::ostream& os, const Value& value, const ExifData* metadata)
+    {
+        if (value.typeId() == Exiv2::unsignedShort || value.typeId() == Exiv2::signedShort)
+        {
+            uint16_t bit   = 0;
+            uint16_t comma = 0;
+            for (uint16_t i = 0; i < value.count(); i++ ) { // for each element in value array
+                uint16_t bits = (uint16_t) value.toLong(i);
+                for (uint16_t b = 0; b < 16; ++b) { // for every bit
+                    if (bits & (1 << b)) {
+                        if ( comma++ ) {
+                            os << ",";
+                        }
+                        os << bit;
+                    }
+                    bit++ ;
+                }
+            }
+            // if no bits are set, print the values (array of zeros)
+            if ( !comma ) printValue(os,value,metadata);
```

Comment:
```suggestion
            if ( !comma ) os << 0;
```

Suggesting this change, otherwise for some files `exiv2 -pt` reports something inconsistent like:
```
Exif.Canon.AFPointsInFocus                   Short       4  0 0 0 0
Exif.Canon.AFPointsSelected                  Short       4  25
```

Here's a test file: 
[AAAA5434.exv.zip](https://github.com/Exiv2/exiv2/files/3514624/AAAA5434.exv.zip)

Reply:
I made it N_("Zero") (localized string "Zero").  That's different from 0 == bit 0 is set!

I've updated the test suite.  I don't see any reason to add your file AAAA5440.exv to the test suite.  However if you think it reveals something not covered by existing 3 test files, please let me know.

Reply:
> I made it N_("Zero") (localized string "Zero").

Sry, but printing a string is even more inconsistent than printing four zeros.

>  That's different from 0 == bit 0 is set!

Ok, so why not changing it to one-based indexing, so that bit 0 yields 1 ?

> I don't see any reason to add your file AAAA5440.exv to the test suite.

AAAA5440.exv has `AFPrimaryPoint` set.